app/src/main/scala/org/alephium/explorer/persistence/schema/CustomGetResult.scala

```diff
@@ -80,6 +80,21 @@ object CustomGetResult {
       U256.unsafe(result.nextBigDecimal().toBigInt.bigInteger)
     }
 
+  @SuppressWarnings(Array("org.wartremover.warts.OptionPartial"))
+  implicit lazy val appStateValueGetResult: GetResult[AppState.Value] =
+    (result: PositionedResult) =>
+      AppState.Value.deserialize(ByteString.fromArrayUnsafe(result.nextBytes())).toOption.get
```

Comment:
Should we replace `.toOption.get` with throwing the `SerdeError` to get actual stack-trace in the logs in case of failure?

```scala
AppState.Value
  .deserialize(ByteString.fromArrayUnsafe(result.nextBytes()))
  .fold(throw _, value => value)
```